src/main/java/edu/cornell/kfs/fp/batch/CorporateBilledCorporatePaidFlatInputFileType.java

```diff
@@ -0,0 +1,57 @@
+package edu.cornell.kfs.fp.batch;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.fp.businessobject.ProcurementCardTransaction;
+
+import edu.cornell.kfs.fp.CuFPConstants;
+import edu.cornell.kfs.fp.CuFPParameterConstants;
+import edu.cornell.kfs.fp.batch.service.CorporateBilledCorporatePaidCreateDocumentService;
+import edu.cornell.kfs.fp.businessobject.CorporateBilledCorporatePaidTransaction;
+import edu.cornell.kfs.fp.businessobject.CorporateBilledCorporatePaidTransactionExtendedAttribute;
+import edu.cornell.kfs.fp.businessobject.ProcurementCardTransactionExtendedAttribute;
+
+public class CorporateBilledCorporatePaidFlatInputFileType extends ProcurementCardFlatInputFileType {
+    protected CorporateBilledCorporatePaidCreateDocumentService corporateBilledCorporatePaidCreateDocumentService;
+    
+    @Override
+    protected ProcurementCardTransaction buildProcurementCardTransactionObject() {
+        return new CorporateBilledCorporatePaidTransaction();
+    }
+    
+    @Override
+    protected ProcurementCardTransactionExtendedAttribute buildProcurementCardTransactionExtendedAttributeObject() {
+        return new CorporateBilledCorporatePaidTransactionExtendedAttribute();
+    }
+    
+    @Override
+    public String getFileTypeIdentifer() {
+        return CuFPConstants.CORPORATE_BILLED_CORPORATE_PAID_FLAT_INPUT_FILE_TYPE;
+    }
+    
+    @Override
+    public String getFileName(String principalName, Object parsedFileContents, String userIdentifier) {
+        String fileName = "cbcp_" + principalName;
```

Comment:
should the defined file prefix be used instead of hard coded "cbcp_"?
At the same time I can see that the code this is based of does the same thing.